python/pymesh/Lattice2D.py

```diff
@@ -32,6 +32,9 @@ def num_vertices(self):
     def num_edges(self):
         return self.__lattice.GetNumEdges();
 
+    def my_lattice(self):
```

Comment:
Do you really need this property? Was it for debugging? It looks to me like this class exists precisely so that you ahve another level of privacy as opposed to exposing the raw c++ wrapped object no?

Reply:
I will get rid of this abstraction altogether.

Reply:
we can get rid of this layer when we do the migration to floor plan generator repo